README.md

```diff
@@ -1,6 +1,6 @@
-# PREMAKE 5 (core) 
-* Linux: [![Build Status](https://travis-ci.org/premake/premake-core.svg?branch=master)](https://travis-ci.org/premake/premake-core)
-* Windows: [![Build Status](http://jenkins.tomvandijck.com/job/premake-core/badge/icon)](http://jenkins.tomvandijck.com/job/premake-core/)
+# PREMAKE 5 (core)
+* Linux  : [![Build Status](https://travis-ci.org/premake/premake-core.svg?branch=master)](https://travis-ci.org/premake/premake-core)
+* Windows: [![Build status](https://ci.appveyor.com/api/projects/status/j34shfmryvqh1c6x?svg=true)](https://ci.appveyor.com/project/tvandijck/premake-core)
```

Comment:
Yes, I was going to email @starkos about that, since he owns the "github organization" only he can make that appveyor page...

I just wanted to get this working, so that we're not relying on my manually operated box somewhere in the cloud.. As much as I love to provide it, it's just work I don't have to do, since there is this thing..

Anyway, if @starkos makes the organization on appveyor, and adds me as an admin, I can set this up correctly...